### PR TITLE
Added dmenu support

### PR DESCRIPTION
**Pull request by paniash**
If the user does not provide any argument, a dmenu prompt is shown to search for anime.

### CODE REVIEW

File: ani-cli [hunk 1]
```diff
@@ -98,7 +109,7 @@ done
 
 shift $((OPTIND - 1))
 
-[ -z "$*" ] && { help_text ; die "Search query not provided";  }
```
Comment (paniash) [inline ani-cli:101]: This line could be entirely removed but it's your call.

**Comment by pystardust** [comment]
Hey @paniash thats nice, but it would be nice if the whole anime selection is also done in dmenu. This would still require the terminal open for selecting anime etc

**Comment by paniash** [comment]
Yeah, makes sense. I'll try adding that too if possible.

**Comment by ura43** [comment]
This seems very stale and has no issue to track the Feature request. Please reimplement and open another PR if the feature is still desired

**Pull request closed without merging**